### PR TITLE
abis/linux: Add some additional inet stuff

### PR DESCRIPTION
**Pull request by Dennisbonke**
2 structs were added to the Linux abi.

Part of the mlibc LFS project.

### CODE REVIEW

File: abis/linux/in.h [hunk 1]
```diff
@@ -39,11 +39,44 @@ struct sockaddr_in6 {
...
 
+struct in_pktinfo {
+	unsigned int ipi_ifindex;
+	struct in_addr ipi_spec_dst;
+	struct in_addr ipi_addr;
+};
+
+struct in6_pktinfo {
+	struct in6_addr ipi6_addr;
+	uint32_t ipi6_ifindex;
```
Comment (no92) [inline abis/linux/in.h:66]: My local Linux headers define this as an `int`. Is this correct?

File: abis/linux/in.h [hunk 2]
```diff
@@ -39,11 +39,44 @@ struct sockaddr_in6 {
...
+	struct in_addr imr_sourceaddr;
+};
+
 struct ipv6_mreq {
 	struct in6_addr ipv6mr_multiaddr;
 	unsigned        ipv6mr_interface;
 };
 
+struct in_pktinfo {
+	unsigned int ipi_ifindex;
```
Comment (no92) [inline abis/linux/in.h:59]: My local Linux headers define this as an int. Is this correct?

File: abis/linux/in.h (hunk 1 above)
Comment (no92) [reply abis/linux/in.h:66]: Is this the correct header to define this? On my local system, this seems to be defined in `<linux/ipv6.h>`

**Comment by Dennisbonke** [comment]
Closing as #737 supersedes this PR.

**Pull request closed without merging**